src/desktop/components/inquiry_questionnaire/map/decisions.coffee

```diff
@@ -7,7 +7,7 @@ decisions =
   ask_specialist: ({ artwork, ask_specialist }) ->
     artwork.get('is_in_auction') is true || ask_specialist is true
 
-  enable_new_inquiry_flow: ({ enableNewInquiryFlow }) -> enableNewInquiryFlow
+  enable_new_inquiry_flow: ({ enableNewInquiryFlow }) -> !!enableNewInquiryFlow
```

Comment:
This was undefined in mobile (this is responsive), and so it wasn't true _or_ false, and thus the decision tree wasn't being evaluated properly.

Reply:
So this same decisions file is used in mobile flow as well?

Reply:
Yes, the inquiry questionnaire/flow is responsive.

The issue is that on desktop you get some stuff bootstrapped from the desktop artwork page (everything in `sd.COMMERCIAL` for example), however when using the `/inquiry/:artwork` route on mobile web you don't have that.